exercises/concept/secure-treasure-chest/.docs/hints.md

```diff
@@ -0,0 +1,25 @@
+todo
+# General
+
+## 1. Define the SecureTreasureChest generic custom type
```

Comment:
Let's add one hint, along the lines of

- Don't forget to also expose other functions required by the tests